includes/admin/class-wc-rest-payments-terminal-locations-controller.php

```diff
@@ -144,18 +144,19 @@ public function get_store_location( $request ) {
 
 		try {
 			// Check the existing locations to see if one of them matches the store.
-			$name = get_bloginfo();
+			// Originally we picked `get_bloginfo` for generating names, but later switched to `site_url` for max immutability.
+			$possible_names = [ get_bloginfo(), site_url() ];
```

Comment:
if `site_url()` returns something like `https://www.example.com`, then maybe for better readability for the merchants to make it `example.com`? Also, can it be that stripe doesn't allow special symbols here?

Otherwise logic looks good to me! Thank you 🙏

Reply:
Good proposal and even less mutability! Addressed in cde996b82a24d9f387bc603667ead6074a659e7a.